### ISSUE TITLE
check_ping & Fedora Core 1/2/3 [sf#1073779]

### ISSUE THREAD

**Issue by monitoring-user** [monitoring-plugins/monitoring-plugins#356]
Submitted by b_eugene on 2004-11-26 16:00:02

./check_ping -H 192.168.1.1 -w 100,20% -c 500,60%
/bin/ping -n -U -w 10 -c 5 192.168.1.1
## Error: Could not interpret output from ping command

I make change in source code of check_ping.c:
437c437
<                                sscanf(buf,"rtt
## min/avg/max/mdev = %_f/%f/%_f/%*f ms",&rta)==1)

> ```
>                            sscanf(buf,"rtt
> ```
> ## min/avg/max/mdev = %f/%f/%f/%f ms",&rta)==1)
> 
> and
> ./check_ping -H 192.168.1.1 -w 100,20% -c 500,60%
> PING OK - Packet loss = 0%, RTA = 0,00 ms
